src/emerge.cpp

```diff
@@ -426,6 +426,10 @@ bool EmergeManager::pushBlockEmergeData(
 				m_qlimit_generate : m_qlimit_diskonly;
 			if (count_peer >= qlimit_peer)
 				return false;
+		} else {
+			// limit block enqueue requests for ABMs to 1/2 of total
```

Comment:
Out of interest: where is the code that request the emerges for `PEER_ID_INEXISTENT`?

Reply:
I should have added that to the comment. It's here: https://github.com/minetest/minetest/blob/master/src/map.cpp#L1698 , called from here: https://github.com/minetest/minetest/blob/master/src/serverenvironment.cpp#L1371

Reply:
Ah, it'd be better if the comment said "requests for active blocks" then since it's not just ABMs (but also objects and nodetimers).

Reply:
Agreed.

Reply:
update comment and commit message